lib/ZumoOta/src/FlashManager.cpp

```diff
@@ -108,37 +108,29 @@ size_t FlashManager::readingStream(uint8_t* expectedResponse, size_t mybytes)
 }
 
 
-bool FlashManager ::sendCommand(const uint8_t* command, size_t commandsize)
+bool FlashManager::sendCommand(const uint8_t* command, size_t commandsize)
 {
+    Stream& deviceStream   = Board::getInstance().getDevice().getStream();
+    if(nullptr == command)
+    {
+        LOG_INFO("command is a nullptr");
+        return false;
+    }
+    else
     {
-        Stream& deviceStream   = Board::getInstance().getDevice().getStream();
-    
-       if(nullptr == command)
-       {
-          LOG_INFO("command is a nullptr");
-          return false;
-       }
-       else
-       {
-       
         /* Send the OpCode and command data to Zumo robot. */
         size_t bytesWritten= deviceStream.write(command, commandsize);
-      
-         LOG_INFO("byteswritten ist %d", bytesWritten);
-    
+        LOG_INFO("byteswritten ist %d", bytesWritten);
         if(bytesWritten == commandsize)

```

Comment:
blank after if

Reply:
It will be taken care of by the next commit.